packages/mui-codemod/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/codemod",
-  "version": "5.2.2",
+  "version": "5.2.8",
```

Comment:
Does it make sense to follow the `Core` version?

Reply:
I think you don't need to update the version in this package since we've been only updating it if the release contains change relevant to the package.